### ISSUE TITLE
The side face seems not good in training phase.

### ISSUE THREAD

**Issue by River-Zhang** [YuliangXiu/ICON#152]
I followed the instruction to process the THuman2.0 dataset and train the ICON. After 10 epochs, the result on tensorboard seems not good especially the reconstruction of the side face. I can't find where the problem is.
![image](https://user-images.githubusercontent.com/60051616/203708025-9b79b88b-36f2-4d25-92e6-6cd78ea8df3e.png)
![image](https://user-images.githubusercontent.com/60051616/203708159-2bd0604f-d887-468e-b67d-a4a0a2dd157f.png)

The change of loss during training is 
 
![微信图片_20221124141912](https://user-images.githubusercontent.com/60051616/203708783-6329fd24-c37c-4121-ae14-22676feb94aa.png)

**Comment by River-Zhang** [comment]
Besides, during infer phase, the generated mesh looks strange from the side. Is this the problem of training?
![image](https://user-images.githubusercontent.com/60051616/203711741-44ea1703-9c4a-4474-9262-37037715ef49.png)
![image](https://user-images.githubusercontent.com/60051616/203711991-eed55241-0026-47ab-b117-66420d4bbe85.png)

**Comment by River-Zhang** [comment]
Or this is because THuman2.0 is not suitable for training?

**Comment by YuliangXiu** [comment]
I highly recommend checking your dataloader first to make sure the SMPL body is well aligned with input image, described in [ICON/training.md](https://github.com/YuliangXiu/ICON/blob/master/docs/training.md#prerequirement)

```python
# visualization for SMPL-X mesh
python -m lib.dataloader_demo -v -c ./configs/train/icon-filter.yaml

# visualization for voxelized SMPL
python -m lib.dataloader_demo -v -c ./configs/train/pamir.yaml
```

**Comment by River-Zhang** [comment]
OK, thanks very much, I'll try this.

**Comment by River-Zhang** [comment]
Wow you are right! I find that the SMPL body is not aligned with the input image, but how can I fix it? Should I scale the SMPL to a smaller size?
![2d2cd6011c945cdf73c49f08531b077](https://user-images.githubusercontent.com/60051616/203910420-bcf289c6-a1f1-442f-94ae-3a5d321c838f.png)

**Comment by YuliangXiu** [comment]
This should not happen if you didn't change my original code. The scale of SMPL should be [-1,1]

Check [this gif](https://github.com/YuliangXiu/ICON/blob/master/docs/training.md#prerequirement) to see the right alignment.

**Comment by River-Zhang** [comment]
yeah, I redownload the smplx data according to training.md and I find that the problem is solved!
![8244f0b8b053b25dfa8973d1b38fed2](https://user-images.githubusercontent.com/60051616/203929865-a5e23067-7f2d-4873-aec2-f036184382d7.png)

Thanks very much!
